src/Controls/samples/Controls.Sample/Pages/Others/BlazorPage.xaml.cs

```diff
@@ -2,10 +2,10 @@
 using Maui.Controls.Sample.Pages.Base;
 using Microsoft.Maui.Controls;
 using Microsoft.Maui.Graphics;
-using Microsoft.AspNetCore.Components.Web;
-using Maui.Controls.Sample.Pages.Others;
 
 #if NET6_0_OR_GREATER
+using Microsoft.AspNetCore.Components.Web;
```

Comment:
This fixes our classic solutions

Reply:
Hmm but this same namespace is also imported a few lines up? How would this be needed?

Reply:
Ah, that makes sense now (you removed from the non-`#if`'ed part. That looks good then!

Reply:
yea I think a rebase or something added them back in :-/ I meant to delete those